src/plextrac

```diff
@@ -237,7 +237,7 @@ function mod_install() {
   title "Installing PlexTrac Instance"
   requires_user_plextrac
   mod_configure
-  mod_check
+  INSTALL_ONLY=1 mod_check
```

Comment:
I think I would just do the direct call to validate the docker syntax here instead of running the whole `mod_check` since `mod_check` by design assumes a running system.

Reply:
Copied logic from mod_check into the install to validate compose file before DBs are spun up.